tests/Tests.AzureAppConfiguration/Integration/IntegrationTests.cs

```diff
@@ -0,0 +1,544 @@
+using Azure;
+using Azure.Core;
+using Azure.Data.AppConfiguration;
+using Azure.Identity;
+using Azure.ResourceManager;
+using Azure.ResourceManager.AppConfiguration;
+using Azure.ResourceManager.AppConfiguration.Models;
+using Azure.ResourceManager.Resources;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration.FeatureManagement;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Tests.AzureAppConfiguration
+{
+    /// <summary>
+    /// Integration tests for Azure App Configuration that connect to a real service.
+    /// Creates a temporary App Configuration store for testing and deletes it after the tests are complete.
+    /// Requires Azure credentials with appropriate permissions.
+    /// </summary>
+    [Trait("Category", "Integration")]
+    [CollectionDefinition(nameof(IntegrationTests), DisableParallelization = true)]
+    public class IntegrationTests : IAsyncLifetime
+    {
+        // Test constants
+        private const string TestKeyPrefix = "IntegrationTest";
+        private const string SentinelKey = TestKeyPrefix + ":Sentinel";
+        private const string FeatureFlagKey = ".appconfig.featureflag/" + TestKeyPrefix + "Feature";
+
+        // Azure Resource Management constants
+        private const string ResourceGroupEnvVar = "AZURE_APPCONFIG_RESOURCE_GROUP";
+        private const string SubscriptionIdEnvVar = "AZURE_SUBSCRIPTION_ID";
+        private const string LocationEnvVar = "AZURE_LOCATION";
+        private const string CreateResourceGroupEnvVar = "AZURE_CREATE_RESOURCE_GROUP";
+        private const string DefaultLocation = "eastus";
+        private const string LocalSettingsFile = "local.settings.json";
+
+        // Keys to create for testing
+        private readonly List<ConfigurationSetting> _testSettings = new List<ConfigurationSetting>
+        {
+            new ConfigurationSetting($"{TestKeyPrefix}:Setting1", "InitialValue1"),
+            new ConfigurationSetting($"{TestKeyPrefix}:Setting2", "InitialValue2"),
+            new ConfigurationSetting(SentinelKey, "Initial"),
+            ConfigurationModelFactory.ConfigurationSetting(
+                FeatureFlagKey,
+                @"{""id"":""" + TestKeyPrefix + @"Feature"",""description"":""Test feature"",""enabled"":false}",
```

Comment:
The `@` looks unnecessary here.

Reply:
When removed it's a compile error, I think it might just be easier to leave as is?

Reply:
Why is it a compilation error

Reply:
The choice is basically to escape each double quote with backslash or with 2 quotes like when we use the `@` symbol, the compile error was just because I didn't escape it correctly the first time. I feel like keeping it this way (using `@`) is more legible since it's consistent with other tests.

Reply:
The purpose of `@` is to avoid having to escape characters that would otherwise require escaping in a regular string. However, if you are saying we are using this elsewhere then I'm fine to keep it.